### DISCUSSION TITLE
Product status #itsgoodnews

### DISCUSSION THREAD

**Discussion by mattelen** [shentao/vue-multiselect#1589] [General]
Hi all

The Vue Mulitselect project has been a bit dormant over the last couple of years, but some eagle eyed devs will notice that that has been changing lately. So as a maintainer team, we want to let you know where we are going with the project, and where to from here.

**One final V2 release**
First up, we are focusing on fixing major and quick bugs related to the current stable version of V2. Once we release this, as this will be the only version of the multiselect that will support Vue 2, we will only be releasing security and major show stopper bugs until Vue 2 reaches End of Life. We expect this release to be done in the coming weeks, so you can keep track of progress with this milestone: https://github.com/shentao/vue-multiselect/milestone/2

**Getting V3 stable and released**
After the final V2 release, we will then bring over those bug fixes to V3, and getting that ready for stable release. This will start with a beta release (https://github.com/shentao/vue-multiselect/milestone/3), as we move towards full stable. We want V3 to be a Vue 3 compatible version of V2, therefore we won't be adding any breaking features into V3, to make it easier for Vue migrations to happen.
With the beta version, we want to make sure we have any major bugs ironed about, and have all our documentation updated, then will release a stable version.

**The future**
Once V3 is stable, we'll then be working on V4. This is where we'll introduce new features, and if needed breaking changes. So if you open a PR that is a major change, it may sit un-reviewed for a little bit until we get to this stage, but we will get there

So, just a brief update on where we want to head, and we hope you'll join us for the journey. If you do find any issues and bugs, please let us know by opening an issue. Also, if you have any feature ideas/requests, start a discussion with your thoughts.

**Comment by massiws** [comment]
Finally a good news: we may hope to view that _v3-alpha_ release go ahead! 
Good job!

**Comment by rbecheras** [comment]
One final V2 release: DONE 🟢  👏  🥳

**Comment by Rubon72** [comment]
is it dead?

**Reply by mattelen** [reply]
No this isn't dead - although fair assumption to make. I've been trying to get the documentation ready for a V3 release - finding the time has been a mission. Anyway, this is now complete, and I'm completing final checks for a V3 release

**Comment by mattelen** [comment]
In case it was missed, we released the stable version of the Vue Multiselect about 9 months ago. Yay. The docs have been updated, so if you haven't already, I recommend checking them out
